src/main/app/src/app/informatie-objecten/informatie-objecten.service.ts

```diff
@@ -117,6 +117,12 @@ export class InformatieObjectenService {
         );
     }
 
+    ondertekenInformatieObject(uuid: string, zaakUuid: string) {
+        return this.http.post<void>(this.addZaakParameter(`${this.basepath}/informatieobject/${uuid}/ondertekenen`, zaakUuid), null).pipe(
```

Comment:
Gebruik onderteken ipv ondertekenen. Hier heb je het wel goed gebruikt in de methodenaam.

Reply:
Hernoemd in nieuwe commit